symfony-cli crashes when docker container are started

Hi there

We are using the symfony cli with docker integration, so we can quickly spin up development dependencies like external web api. 

In our compose.yml we define the external dependencies like so:

```yaml
  webservice:
    image: mockoon/cli:latest
    command: ["--data", "/data.json", "--port", "3000"]
    labels:
      com.symfony.server.service-prefix: 'UB_WEBSERVICE'
    healthcheck:
      test: ["CMD-SHELL", "curl -f http://localhost:3000/_health || exit 1"]
      interval: 10s
      timeout: 5s
    ports: [ 3000 ]
    volumes:
      - ./.devcontainer/docker/mockoon.json:/data.json:readonly
```

since the release of 5.10, the symfony cli crashes when you type any subcommand while the docker containers are running:

```console
$ symfony console
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/symfony-cli/symfony-cli/envs.extractRelationshipsEnvs({0xcddb60, 0xc00033c720})
        github.com/symfony-cli/symfony-cli/envs/envs.go:127 +0x8c38
github.com/symfony-cli/symfony-cli/envs.AsMap({0xcddb60, 0xc00033c720})
        github.com/symfony-cli/symfony-cli/envs/envs.go:80 +0xc5
github.com/symfony-cli/symfony-cli/local/php.(*Executor).Config(0xc00016e420, 0x0)
        github.com/symfony-cli/symfony-cli/local/php/executor.go:182 +0x18e
github.com/symfony-cli/symfony-cli/local/php.(*Executor).Execute(0xc00016e420, 0xfe?)
        github.com/symfony-cli/symfony-cli/local/php/executor.go:311 +0x45
main.main()
        github.com/symfony-cli/symfony-cli/main.go:78 +0x7e9
```

If the containers are not started, no panic is thrown. 

The line `github.com/symfony-cli/symfony-cli/envs/envs.go:127` suggests that it wants to parse the scheme for the new database ha feature. This should not be applied on this container (in this project we do not even have a database).

Steps to reproduce:
- Download latest symfony cli
- Create new project `symfony new --webapp test`
- Create any service definition with a service prefix in compose.yml 
- Execute a subcommand in symfony cli, e.g. `symfony console`

--- a/envs/docker.go
+++ b/envs/docker.go
@@ -460,6 +460,8 @@ func (l *Local) dockerServiceToRelationship(client *docker.Client, container typ
 			rels[""]["scheme"] = "http"
 		} else if p.PrivatePort == 443 || p.PrivatePort == 8443 {
 			rels[""]["scheme"] = "https"
+		} else {
+			return nil
 		}
 		return rels
 	}

